mappings/net/minecraft/client/render/BufferBuilder.mapping

```diff
@@ -6,3 +6,20 @@ CLASS net/minecraft/class_1002 net/minecraft/client/render/BufferBuilder
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
+	METHOD method_9550 draw ()I
+	METHOD method_9552 uv (DD)V
+		ARG 1 u
+		ARG 3 v
+	METHOD method_9553 vertex (DDD)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+	METHOD method_9554 vertexUv (DDDDD)V
```

Comment:
``vertexUV``

Reply:
upstream calls it `uv`
see `BufferBuilder`'s `light(int)` and `overlay(int)` methods

Reply:
upstream calls it `uv`
see `BufferBuilder`'s `light(int)` and `overlay(int)` methods